src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -915,6 +915,7 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
               f'{key} = {str(value).encode("unicode-escape").decode("utf-8")}\n'
           )
 
+      job.requires_gpu = bool(job.get_environment().get('REQUIRES_GPU'))
```

Comment:
As I just discovered earlier, this file is used to sync config files to the database. Was the intent to set the `requires_gpu` field on jobs when UI changes are made to set the `REQUIRES_GPU`?

Reply:
I removed this part. Indeed, I was thinking about a way to update a field in the job. Not needed for the moment, would be useful if we end up adding a UI option